packages/clay-autocomplete/src/Autocomplete.tsx

```diff
@@ -217,7 +228,9 @@ function AutocompleteInner<T extends Item>(
 		defaultItems,
 		defaultValue,
 		direction = 'bottom',
+		displayType,
 		filterKey,
+		insetItemBefore,
```

Comment:
Hey @pat270 we don't want to add new props to the autocomplete for this. We just need to update how the input can be rendered. The autocomplete and other components like it let us pass our own component to be used as the input by using the `As` prop.

Reply:
Great! So we shouldn’t need any changes to autocomplete… likely only changes to input css and maybe some other utility classes.